components/Slider/src/MDCSlider.m

```diff
@@ -229,6 +251,14 @@ - (void)updateColorsForState {
   _thumbTrack.trackOffTickColor = [self backgroundTrackTickColorForState:self.state];
 }
 
+- (void)updateThumbForState {
+  if (!self.isStatefulAPIEnabled) {
+    return;
+  }
```

Comment:
I feel as though if the stateful API is disabled, we should restore the ThumbTrack's thumbRadius to the `-thumbRadius` property value.  What do you think?

Reply:
Done

Reply:
Given what you needed to do (affect the new API from the old API), I think this might have been a mistake. I was thinking that an iVar could be used, but it may just be better to go back to the original version.  When we spoke offline about why a client would be turning the API on and off, I think you made a good point.  We can probably expect clients to only go from the legacy to the new API and not back again.